easybuild/easyconfigs/p/PETSc/PETSc-3.7.3-foss-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,32 @@
+name = 'PETSc'
+version = '3.7.2'
```

Comment:
I retriggered the tests, and now they correctly find this problem, I have no idea why they didn't last time, weird...